### ISSUE TITLE
Points of removed series do not get removed from cartesian chart.

### ISSUE THREAD

**Issue by besentv** [Live-Charts/Live-Charts#819]
#### How to reproduce?

I have a cartesian chart docked inside a panel. When I try to remove one of it's series with all it's points the points will stay in kind of an unusable way: You can't hover over them nor are they still in the Value list.

I use this code to remove a series: 

`                        foreach(Series s in chart.Series)
                        {
                            s.Values.Clear();
                            s.Erase();
                            chart.Series.Remove(s);
                        }`
Before the series get removed:

![screenshot_1](https://user-images.githubusercontent.com/10588728/42176685-103eb758-7e2a-11e8-923f-fcfe18d79995.png)

After the series gets removed:

![screenshot_2](https://user-images.githubusercontent.com/10588728/42176697-25558040-7e2a-11e8-88c7-a67c296ac053.png)

#### Extra notes

How can I fix this in such a way the points won't show up? And how can I make the chart use the colors of the previous series again?

#### I am using

<table>
<tr>
    <td>Live-Charts version</td>
    <td>0.9.7</td>
</tr>
<tr>
    <td>.Net Version</td>
    <td>4.5</td>
</tr>
<tr>
    <td>Windows</td>
    <td>8.1 x64 Pro</td>
</tr>
</table>

**Comment by tmbarker** [comment]
@besentv, I am experiencing the same issue. Did you ever find a workaround/solution?

**Comment by LabuzzMichal** [comment]
If you want do remove whole serie you might simply hide it (setting visibility) instead of removing. That works for sure, however I know it is really poor workaround

**Comment by tmbarker** [comment]
Thanks for the suggestion, but as you noted this is a pretty poor work-around, and it "contaminates" other sections of my code, like my legend control, etc.

What's strange about this is that it's not clear to me exactly what causes it. I have other applications which call 'Clear() ' on LineSeries objects bound to Cartesian charts and the point geometries are erased just fine. Perhaps if someone can explain what the conditions are exactly to trigger this bug I can simply avoid them

**Comment by LabuzzMichal** [comment]
Did you try to set null on PointGeometry  before clearing serie?

**Comment by tmbarker** [comment]
Yes, this did not work for me, and neither did setting the PointGeometrySize to 0 before clearing the series. 

Curiously enough, if I set the PointGeometrySize to 0 in the constructor of the LineSeries object then the series does clear properly. It's a decent work-around, of course with the draw back of having no data markers on your points, which obviously could be very important depending on the application. Unfortunately for my application I would really like to have data markers.
